rest-api/test/unit_test/api_test/participant_api_test.py

```diff
@@ -111,9 +111,20 @@ def test_change_pairing_for_org_then_site(self):
     participant = self.send_post('Participant', self.participant)
     participant['providerLink'] = [ self.provider_link_2]
     participant_id = participant['participantId']
-    participant['organization'] = 'AZ_TUCSON_BANNER_HEALTH'
     path = 'Participant/%s' % participant_id
-    update_awardee = self.send_put(path, participant, headers={'If-Match': 'W/"1"'})
-    update_awardee['site'] = 'hpo-site-bannerphoenix'
-    self.send_put(path, update_awardee, headers={'If-Match': 'W/"2"'})
-    self.assertEqual(update_awardee['site'], 'hpo-site-bannerphoenix')
+
+    update_1 = self.send_put(path, participant, headers={'If-Match': 'W/"1"'})
+    participant['site'] = 'hpo-site-bannerphoenix'
+    update_2 = self.send_put(path, participant, headers={'If-Match': 'W/"2"'})
+    self.assertEqual(update_1['site'], 'UNSET')
+    self.assertEqual(update_1['organization'], 'UNSET')
+    self.assertEqual(update_2['site'], 'hpo-site-bannerphoenix')
+    self.assertEqual(update_2['organization'], 'UNSET')
```

Comment:
This looks wrong. If site is sent for pairing, organization and awardee should be set. If organization is set, awardee should be set (but  site should be UNSET.)

Add more tests and fix behavior?